### PR TITLE
Task/Add test suite

### PR DESCRIPTION
**Pull request by jope1**
Initial testing suite providing framework and initial set of test files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,12 +87,28 @@ source $HOME/pythonvirtenvs/capacity-status-env/bin/activate
 Now run the following to start the application in the root directory of the
 project:
 
+make project-start
 make api-start
 
-The application is configured to run on https://localhost:8443/apidoc/
+The application is configured to run on http://localhost:8000/apidoc/
```
Comment (mabe13) [inline README.md]: Might want to clearly specifiy that this is for development

File: README.md [hunk 2]
```diff
@@ -86,12 +87,28 @@ source $HOME/pythonvirtenvs/capacity-status-env/bin/activate
...
+* Ensure test files are created in a relevant sub-directory under the test directory of the app under test.
+* Ensure all test files are prefixed with "test_" and match the name of the module/class that this test file tests.
+* Ensure that import unittest is brought into the test file.
+* Base your test class on unittest.TestCase.
+* Add the new test file to the __init__.py file so that the unit test runner picks the new tests up.
+
+### Running the unit tests
+
+The unit tests can be run by executing the following command:
+    ./manage.py test app
```
Comment (mabe13) [inline README.md]: I think the run here needs to be ./application/manage.py test api

File: README.md (hunk 1 above)
Comment (jope1) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (jope1) [reply README.md]: Actioned.

**Pull request merged**